src/util/fs.js

```diff
@@ -511,6 +734,22 @@ export async function writeFilePreservingEol(path: string, data: string) : Promi
   await promisify(fs.writeFile)(path, data);
 }
 
+export async function hardlinksWork(cwd: string): Promise<boolean> {
```

Comment:
Maybe just call the param `dir`, as it could be called with any directory - It doesn't *have* to be the current directory.